.ci/packaging.groovy

```diff
@@ -211,6 +211,8 @@ def linuxPlatforms() {
             'windows/amd64',
             'windows/386',
             'darwin/amd64'
+            // TODO(AndersonQ): comment in after the tests pass
+            // 'darwin/arm64'
```

Comment:
What is the plan here? Comment it in once CI is green and merge or do it in another PR?

Reply:
I ended up testing it on the elastic-agent repo, didn't do what I needed. Besides in order to unblock the release team, better to merge it now and adjust the CI later. But good catch anyway :)